app/views/patients/show.html.erb

```diff
@@ -61,7 +61,7 @@
             <th class="DataTable-table-header">Reporter</th>
             <th class="DataTable-table-header">Created</th>
             <% columns.each do |column| -%>
-              <th class="DataTable-table-header"><%= column.titleize %></th>
+              <th class="DataTable-table-header"><%= column&.titleize %></th>
```

Comment:
Would it be better to filter out these columns in the first place? Add `.reject(&:nil?)` to columns above

Reply:
Since the columns are based on the Threshold_condition now there should never be nil columns. However, if there somehow was, this would result in a blank column header with assessment responses filled out in that column for any reported condition that for whatever reason ended up with a nil name... so we either show a blank column header with values populated in the associated columns of the assessments or don't show the blank column header and throw out any of the reported symptoms that were constructed from the threshold condition that didn't have a name for whatever reason.

Reply:
I see, in that case I think this is the way to go, especially since removing the blanks could result in mismatched td/th counts